spec/swagger.yaml

```diff
@@ -114,7 +114,7 @@ parameters:
   resourceId:
     name: id
     in: path
-    description: The resource identifier string
+    description: Checkout page custom url path segment
```

Comment:
FYI, this description makes no sense, since it is overridden in `CheckoutPage`

Reply:
It overriden only for payload. This is param description. You can check rendered result.

Reply:
You can override the param description as well, not here in shared definition.